project/project.go

```diff
@@ -6,6 +6,8 @@ import (
 	"os/exec"
 	"path/filepath"
 
+	lua "github.com/yuin/gopher-lua"
```

Comment:
I suggest to place "lua" logic into separate helper and require "lua" package only in such place.

Reply:
Moved into `common`.